render/src/bounding_box.rs

```diff
@@ -11,6 +11,14 @@ pub struct BoundingBox {
 }
 
 impl BoundingBox {
+    /// Create a zero-sized, valid BoundingBox
+    pub fn empty() -> Self {
```

Comment:
How about make a custom impl Default that sets valid to true?
As an all-zero BoundingBox should be implicitly valid.

Reply:
I tried doing this originally, however it causes some of our tests to fail. (Also this change isn't needed now that avm2 buttons overload `bounds_with_transform`)

Reply:
> however it causes some of our tests to fail

Oh, that's even weirder I think. Recently I was thinking that `valid` is actually redundant and just equivalent to doing validity check manually each time.